dts/Kconfig

```diff
@@ -19,3 +19,10 @@ config HAS_DTS_I2C
 	This option specifies that the target platform supports device tree
 	configuration for I2c.
 
+config HAS_DTS_SENSOR
```

Comment:
I think this should now be `HAS_DTS_I2C_DEVICE`?

Reply:
Good catch!